mmhuman3d/utils/ffmpeg_utils.py

```diff
@@ -1041,7 +1047,8 @@ def slice_video(input_path: str,
         input_path (str): input video or gif file path.
         output_path (str): output video of gif file path.
         start (int, optional): start frame index. Defaults to 0.
-        end (int, optional): end frame index. Defaults to -1.
+        end (int, optional): end frame index. Excluded.
+            Defaults to None.
```

Comment:
Similar as above.

Reply:
in ffmpeg, the `end_frame` must be positive